samples/ORKCatalog/ORKCatalog/Resources/Localized/Base.lproj/LaunchScreen.xib

```diff
@@ -25,10 +28,12 @@
                 </navigationBar>
                 <tabBar contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="pMQ-9U-JXX">
                     <rect key="frame" x="0.0" y="431" width="480" height="49"/>
+                    <animations/>
                     <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="calibratedWhite"/>
                     <items/>
                 </tabBar>
             </subviews>
+            <animations/>
             <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="calibratedWhite"/>
             <constraints>
                 <constraint firstItem="pv7-RZ-bDW" firstAttribute="top" secondItem="KFe-cA-LWE" secondAttribute="bottom" id="ERQ-ec-2pG"/>
```

Comment:
No - I hadn't changed anything here at all! Would Xcode have changed this (I had Xcode 7.1 Beta on my laptop - I note that the editor version was different a few lines higher up. Would any kind of automatic change have been made if it had loaded and saved the file for some reason?